Content.Server/GameTicking/GameTicker.Spawning.cs

```diff
@@ -55,7 +55,9 @@ private List<EntityUid> GetSpawnableStations()
             return spawnableStations;
         }
 
-        private void SpawnPlayers(List<ICommonSession> readyPlayers, Dictionary<NetUserId, HumanoidCharacterProfile> profiles, bool force)
+        private void SpawnPlayers(List<ICommonSession> readyPlayers,
+            Dictionary<NetUserId, HumanoidCharacterProfile> profiles,
+            bool force)
```

Comment:
This does not match what's in the conventions doc. The first argument should be on its own line.

Reply:
It looks like there's a reshaper option that does this, `resharper_csharp_wrap_after_declaration_lpar`

Reply:
Ah yeah, I wasn't sure whether to add it. Feel free to make a new PR with it, or I'll get to it tomorrow when I don't have PR reviews checked out.